lib/samples/add_wms_layer/add_wms_layer.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps_sdk/arcgis_maps.dart';
+import 'package:flutter/material.dart';
+
+class AddWmsLayer extends StatefulWidget {
+  const AddWmsLayer({super.key});
+
+  @override
+  State<AddWmsLayer> createState() => _AddWmsLayerState();
+}
+
+class _AddWmsLayerState extends State<AddWmsLayer> {
```

Comment:
Should we always do `with SampleStateSupport` even for samples that don't call setState? I think so, because the danger is we'll forget it when we need it, but not get any warning.

Reply:
Yeah fair enough, good spot. This wasn't deliberate. I use the `stfl` shortcut to create widgets and just forgot about it.